packages/react-aria-components/stories/Button.stories.tsx

```diff
@@ -25,6 +27,58 @@ export const ButtonExample = () => {
   );
 };
 
+export const PendingButton = {
+  render: (args) => <PendingButtonExample {...args} />,
+  args: {
+    children: 'Press me'
+  }
+};
+
+function PendingButtonExample(props) {
+  let [isPending, setPending] = useState(false);
+
+  let timeout = useRef<ReturnType<typeof setTimeout> | undefined>(undefined);
+  let handlePress = (e) => {
+    action('pressed')(e);
+    setPending(true);
+    timeout.current = setTimeout(() => {
+      setPending(false);
+      timeout.current = undefined;
+    }, 5000);
+  };
+
+  useEffect(() => {
+    return () => {
+      clearTimeout(timeout.current);
+    };
+  }, []);
+
+  return (
+    <Button
+      {...props}
+      isPending={isPending}
+      onPress={handlePress}
+      className={styles2['button']}>
+      {({isPending}) => (
+        <>
+          <Text className={isPending ? styles2['pending'] : undefined}>{props.children}</Text>
+          <ProgressBar
+            aria-label="loading"
```

Comment:
Why are we using "loading" as the label when it's "pending" in RSP and the feature is called pending?

Reply:
This is just a story, and I think I had pending as a default at one point, wanted something different.